app/policies/project_policy.rb

```diff
@@ -98,9 +98,37 @@ def update_sample?
     false
   end
 
+  def transfer_sample?
+    return true if record.namespace.parent.user_namespace? && record.namespace.parent.owner == user
+    return true if Member.can_transfer?(user, record.namespace) == true
+
+    details[:name] = record.name
+    false
+  end
+
+  def transfer_sample_into_project?
+    return true if record.namespace.parent.user_namespace? && record.namespace.parent.owner == user
+    return true if Member.can_transfer?(user, record.namespace) == true
+
+    details[:name] = record.name
+    false
+  end
+
   scope_for :relation do |relation|
     relation.where(namespace: { parent: user.groups.self_and_descendant_ids })
             .include_route.order(updated_at: :desc).or(relation.where(namespace: { parent: user.namespace })
             .include_route.order(updated_at: :desc))
   end
+
+  scope_for :relation, :transferable do |relation|
```

Comment:
Please see 1c68e9870cf52381e354459c388f187d2a219660.

1. True, but I'm terrible at naming things. I don't think `sample_transferable` is any better. I know you've used `manageable` for namespaces. I was worried that might be confusing, but this is suppose to list projects which the user is an owner or maintainer of. 
2. Fixed.

Reply:
Oh don't worry I'm horrible with naming stuff too! I think `manageable` would make sense if we were getting all the projects that the user is an owner and maintainer of. Since we are getting n-1 I think we need another name. I'm going to think on a name and we can discuss it tomorrow!

Reply:
I think I got a name for the scope, `receivable`. Let me know what you think or if you have any other potential names

Reply:
Ok, will do.

Reply:
Hey Katherine, sorry let's change this scope to `manageable` since I missed that we weren't actually removing the current project from the listing. From wherever the scope is being called you can add a `where.not(id: ID_OF_NAMESPACE)` which should remove it from the listing of projects that the sample(s) can be transferred to

Reply:
Please see 928caa1d31798becc289afd1f333959cb00d3621. This branch doesn't have where the scope is being called, but I did update it in the transfer-samples branch.

Reply:
Thanks!